x/staking/keeper/params.go

```diff
@@ -54,3 +54,8 @@ func (k Keeper) MinCommissionRate(ctx context.Context) (math.LegacyDec, error) {
 	params, err := k.Params.Get(ctx)
 	return params.MinCommissionRate, err
 }
+
+func (k Keeper) KeyRotationFee(ctx sdk.Context) (sdk.Coin, error) {
```

Comment:
is this needed since k.params.Get().KeyRotationFee does this same?

Reply:
Yeah, we can do that.